buildenv/jenkins/variables/defaults.yml

```diff
@@ -69,13 +69,12 @@ test_dependencies_job_name: 'test.getDependency'
 slack_channel: '#jenkins'
 build_discarder:
   logs:
-    build: 20
-    pipeline: 20
-    pullRequest: 50
-  artifacts:
-    build: 10
-    pipeline: 0
-    pullRequest: 0
+    Nightly: 10
+    Release: 10
+    OMR: 20
+    OpenJDK: 20
+    Personal: 50
+    Pipeline: 100
```

Comment:
@pshipton are these numbers reasonable to start with?

Reply:
Note: OpenJDK would be for the OpenJDK Acceptance build if/when it is created.

Reply:
And Personal is for if/when personal build are created as well.  Those would come in handy for testing changes like this. :)

Reply:
`Personal` is the default, so anything sandbox or custom and also PullRequests.